packages/access-api/src/service/index.js

```diff
@@ -181,6 +198,18 @@ export function service(ctx) {
       fail() {
         throw new Error('test fail')
       },
+      /**
+       * @param {Ucanto.Invocation<Ucanto.Capability<'testing/space-storage', Ucanto.DID<'key'>, Ucanto.Failure>>} invocation
+       */
+      'space-storage': async (invocation) => {
```

Comment:
yeah. I thought about it but didn't want to extend the public api without discussion.

In this case I am very supportive because that will also be helpful to upload-api and things like https://github.com/web3-storage/w3infra/issues/134#user-content-implementation

Reply:
can I do it as a fast follow once I have verified that the space registration via `provider/add` works on staging at all?
https://github.com/web3-storage/w3protocol/issues/480